api/krusty/testdata/fnmounts/kustomization.yaml

```diff
@@ -0,0 +1,20 @@
+# Test file for TestFnContainerMounts
```

Comment:
Trying to copy the charts directory into the temporary directory where the test harness was running was a bit of a headache for mac vs linux, so I instead created a test kustomization file here so that the test can run `kustomize build` directly on it.